CRW-EthOnline-smart-contract/contracts/CRWCP.sol

```diff
@@ -1,15 +1,11 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
-// any specific tokenomics? max supply? only certain restaurants can mint?
 // what kind of validation do we need to put on the verify function? restaurants can't assign to themselves; whitelist to who the token can be sent
-// should the mint only be called internally from the verify function
```

Comment:
not sure lets discuss

Reply:
it's more implementation detail. This was solved already